cmd/main.go

```diff
@@ -95,7 +95,8 @@ func main() {
 	flag.StringVar(&metricsCertKey, "metrics-cert-key", "tls.key", "The name of the metrics server key file.")
 	flag.BoolVar(&enableHTTP2, "enable-http2", false,
 		"If set, HTTP/2 will be enabled for the metrics and webhook servers")
-	flag.DurationVar(&requeuePeriod, "requeue-period", 15*time.Second, "The default reconciliation requeue period for all Humio* resources.")
+	flag.DurationVar(&requeuePeriod, "requeue-period", 15*time.Second,
```

Comment:
One thing I noticed, after sending this off for review, was that I used the <int>*`time.Duration` style whereas the rest of the code uses `time.Duration*<int>` style. Not a big deal but just thought I would call it out in case you want to align it with the rest of the code.